beta-src/src/utils/state/gameApiSlice/extraReducers/fetchGameData/precomputeLegalOrders.ts

```diff
@@ -0,0 +1,524 @@
+import provincesMapData from "../../../../../data/map/ProvincesMapData";
```

Comment:
I love that this is all encapsulated in one place, so if there's a bug in, say, convoys, there's only one function to look at. Instead of a maze of functions and classes.